src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Confirm.tsx

```diff
@@ -167,12 +176,23 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
     selectPortfolioCryptoBalanceBaseUnitByFilter(s, feeAssetBalanceFilter),
   )
 
+  const amountAvailableCryptoPrecision = useMemo(() => {
+    return bnOrZero(opportunityData?.stakedAmountCryptoBaseUnit)
+      .plus(bnOrZero(opportunityData?.rewardsCryptoBaseUnit?.amounts[0])) // Savers rewards are denominated in a single asset
```

Comment:
```suggestion
      .plus(bnOrZero(opportunityData?.rewardsCryptoBaseUnit?.amounts[0])) // accrued RUNEPool rewards are denominated in RUNE
```

Reply:
Changed in [1f9d825](https://github.com/shapeshift/web/pull/7449/commits/1f9d8254294f6843bde971347039e344909ac032)